core/src/main/java/org/frankframework/pipes/XmlIf.java

```diff
@@ -40,11 +40,10 @@
 /**
  * Selects a forward, based on XPath evaluation
  *
+ * @author Peter Leeuwenburgh
```

Comment:
Staat in ons style-ding dat de auteur bovenaan komt? Die zou ik liever onderaan willen zien, zodat alle 'frank' documentatie gegroepeerd is.

Reply:
Poeh dat zou best kunnen dat IntelliJ probeert om tags in JavaDoc te sorteren... Ik zal eens kijken of dat geconfigureerd kan worden.

Reply:
Ik kan het nergens terugvinden in de settings / .editorconfig. Ik zal het terugdraaien.